packages/@coorpacademy-player-services/src/recommendations.js

```diff
@@ -1,20 +1,31 @@
-const find = fixtures => (type, ref) => {
+// @flow
+
+import type {Fixtures, Level, Recommendation} from './types';
+
+const find = (fixtures: Fixtures) => (
+  type: string,
```

Comment:
COMMENT: should this one be `chapter | level` too ? as the function below

Reply:
added type `ContentType`